README.md

```diff
@@ -567,30 +574,49 @@ To display in-app forms, add the following code to your application.
         .initialize(with: "YOUR_KLAVIYO_PUBLIC_API_KEY")
```

Comment:
Should we remove the line above which says `// Refer the behavior section for where to place this code.` and place the code in `application(_:didFinishLaunchingWithOptions:)`